db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -3502,6 +3503,14 @@ public void clearSPSPlans() throws StandardException{
 
         while(sc.fetchNext(rowTemplate)){
             /* Replace the column in the table */
+            for (int i=0; i<rowTemplate.length; i++) {
+                if (i  == SYSSTATEMENTSRowFactory.SYSSTATEMENTS_VALID-1)
+                    replaceRow[i] = new SQLBoolean(false);
+                else if (i == SYSSTATEMENTSRowFactory.SYSSTATEMENTS_CONSTANTSTATE-1)
```

Comment:
rest of file uses style `i+1 == SYSSTATEMENTSRowFactory.SYSSTATEMENTS_CONSTANTSTATE`, i think we should stick to one style (same line 3507)

Reply:
I made the change to make the style consistent. Thanks!